src/App.tsx

```diff
@@ -1,93 +1,61 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Footer from './components/Footer';
+import NewTodoField from './components/NewTodoField';
+import TodoList from './components/TodoList';
+import { ToggleAllButton } from './components/ToggleAllButton/ToggleAllButton';
```

Comment:
```suggestion
import { ToggleAllButton } from './components/ToggleAllButton';
```
you have index file there